virtual_vehicle_state_topic_name: /virtual_vehicle_state

replay_trigger_topic_name: /control/replay_trigger

init_distance: 5 #[m] the initial distance that virtual vehicle ahead ego vehicle 
